contracts/incentive/IncentiveLayer.sol

```diff
@@ -512,28 +512,27 @@ contract IncentiveLayer is JackpotManager, DepositsManager, RewardsManager {
             s.solution0Challengers.length = 0;
         }
 
-        if (isForcedError(originalRandomBits)) { // this if statement will make this function tricky to test
+        if (isForcedError(t.randomBits, t.blockhash)) { // this if statement will make this function tricky to test
             rewardJackpot(taskID);
-            t.finalityCode = 2;
```

Comment:
Do we want the task to continue as normal if there is a forced error? I thought we were throwing away the task

Reply:
Not sure. But it is more simple if there is no special path. And I cannot see any reason to throw it away.